duecredit/injections/mod_scipy.py

```diff
@@ -25,4 +25,130 @@ def inject(injector):
+                 min_version='v0.4.3')
+
+    injector.add('scipy.cluster.hierarchy', None, BibTeX("""
+    @article{batagelj1995comparing,
+        title={Comparing resemblance measures},
+        author={Batagelj, Vladimir and Bren, Matevz},
+        journal={Journal of classification},
+        volume={12},
+        number={1},
+        pages={73--90},
+        year={1995},
+        publisher={Springer}
+    }"""), description="Global references for the module",
+                 min_version='v0.4.3')
+
+    injector.add('scipy.cluster.hierarchy', None, BibTeX("""
+    @book{sokal1958statistical,
+        author = {Sokal, R R and Michener, C D and {University of Kansas}},
+        title = {{A Statistical Method for Evaluating Systematic Relationships}},
+        publisher = {University of Kansas},
+        year = {1958},
+        series = {University of Kansas science bulletin}
+    }"""), description="Global references for the module",
+                 min_version='v0.4.3')
+
+    injector.add('scipy.cluster.hierarchy', None, BibTeX("""
+    @article{edelbrock1979mixture,
+        title={Mixture model tests of hierarchical clustering algorithms:
+            the problem of classifying everybody},
+        author={Edelbrock, Craig},
+        journal={Multivariate Behavioral Research},
+        volume={14},
+        number={3},
+        pages={367--384},
+        year={1979},
+        publisher={Taylor \& Francis}
+    }"""), description="Global references for the module",
+                 min_version='v0.4.3')
+
+    injector.add('scipy.cluster.hierarchy', None, BibTeX("""
+    @book{jain1988algorithms,
+        title={Algorithms for clustering data},
+        author={Jain, Anil K and Dubes, Richard C},
+        year={1988},
+        publisher={Prentice-Hall, Inc.}
+    }"""), description="Global references for the module",
+                 min_version='v0.4.3')
+
+    injector.add('scipy.cluster.hierarchy', None, BibTeX("""
+    @article{fisher1936use,
+        title={The use of multiple measurements in taxonomic problems},
+        author={Fisher, Ronald A},
+        journal={Annals of eugenics},
+        volume={7},
+        number={2},
+        pages={179--188},
+        year={1936},
+        publisher={Wiley Online Library}
+    }"""), description="Global references for the module",
+                 min_version='v0.4.3')
+    # Here options for linkage
+    injector.add('scipy.cluster.hierarchy', 'linkage', BibTeX("""
+    @article{ward1963hierarchical,
+        title={Hierarchical grouping to optimize an objective function},
+        author={Ward Jr, Joe H},
+        journal={Journal of the American statistical association},
+        volume={58},
+        number={301},
+        pages={236--244},
+        year={1963},
+        publisher={Taylor \& Francis}
+    }"""), conditions={(1, 'method'): {'ward'}},
+                 description="Ward hierarchical clustering",
+                 min_version='v0.4.3')
```

Comment:
I guess this is an example for ```tags=['implementation']``` which is the default, thus ok to not have explicitely listed here